Developer-Guide.md

```diff
@@ -116,11 +135,13 @@ $ go get -d -u github.com/kata-containers/osbuilder
 ```
 
 ## Create a rootfs image
-
+### Create a local rootfs
 ```
 $ cd $GOPATH/src/github.com/kata-containers/osbuilder/rootfs-builder
-$ script -fec 'sudo -E GOPATH=$GOPATH USE_DOCKER=true ./rootfs.sh clearlinux'
+$ script -fec 'sudo -E GOPATH=$GOPATH USE_DOCKER=true ./rootfs.sh ${distro}'
 ```
+`${distro}` can be one of `alpine`, `centos`, `clearlinux`, `euleros` and `fedora`. When unset, it builds
```

Comment:
Lines 143-144 suggested rewrite: You can choose `alpine`, `centos`, `clearlinux`, `euleros`, or `fedora` for `${distro}`. If `${distro}` is not set, it builds 'centos' based rootfs by default.